src/panels/shopping-list/ha-panel-shopping-list.ts

```diff
@@ -0,0 +1,130 @@
+import "@polymer/app-layout/app-header/app-header";
+import "@polymer/app-layout/app-toolbar/app-toolbar";
+import {
+  css,
+  CSSResultArray,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+import memoizeOne from "memoize-one";
+import { mdiMicrophone } from "@mdi/js";
+
+import { HomeAssistant } from "../../types";
+import { haStyle } from "../../resources/styles";
+import { createCardElement } from "../lovelace/create-element/create-card-element";
+import { LovelaceCard } from "../lovelace/types";
+import { HuiErrorCard } from "../lovelace/cards/hui-error-card";
+import { isComponentLoaded } from "../../common/config/is_component_loaded";
+import { showVoiceCommandDialog } from "../../dialogs/voice-command-dialog/show-ha-voice-command-dialog";
+
+import "../../components/ha-menu-button";
+import "../../layouts/ha-app-layout";
+
+@customElement("ha-panel-shopping-list")
+class PanelShoppingList extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ type: Boolean, reflect: true }) public narrow!: boolean;
+
+  @internalProperty() private _card!: LovelaceCard | HuiErrorCard;
+
+  private _conversation = memoizeOne((_components) =>
+    isComponentLoaded(this.hass, "conversation")
+  );
+
+  protected firstUpdated(changedProperties: PropertyValues): void {
+    super.firstUpdated(changedProperties);
+
+    this._card = createCardElement({ type: "shopping-list" }) as LovelaceCard;
+    this._card.hass = this.hass;
+  }
+
+  protected updated(changedProperties: PropertyValues): void {
+    super.updated(changedProperties);
+
+    if (changedProperties.has("hass")) {
+      this._card.hass = this.hass;
+    }
+  }
+
+  protected render(): TemplateResult {
+    return html`
+      <ha-app-layout>
+        <app-header fixed slot="header">
+          <app-toolbar>
+            <ha-menu-button
+              .hass=${this.hass}
+              .narrow=${this.narrow}
+            ></ha-menu-button>
+            <div main-title>${this.hass.localize("panel.shopping_list")}</div>
+            ${this._conversation(this.hass.config.components)
+              ? html`
+                  <mwc-icon-button
+                    .label=${this.hass!.localize(
+                      "ui.panel.lovelace.menu.start_conversation"
```

Comment:
We can't use translation tags from other panels, as they will not be available until that panel is loaded.

Reply:
Will that effect the translations for the shopping list card itself as it does use lovelace translations in it?